app/views/provider_interface/provider_users/edit_providers.html.erb

```diff
@@ -20,6 +20,8 @@
                   <%= ppf.hidden_field :provider_id %>
                   <%= ppf.govuk_check_boxes_fieldset :permissions, legend: { text: 'Choose permissions', size: 's' } do %>
                     <%= ppf.govuk_check_box :manage_users, true, multiple: false, label: { text: 'Manage users' } %>
+                    <%= ppf.govuk_check_box :view_safeguarding_information, true, multiple: false,
```

Comment:
What does `multiple:` mean here?

Reply:
https://govuk-form-builder.netlify.app/form-elements/checkboxes/#generating-a-single-checkbox
I think because this isn't a multiple selection on the same attribute it's necessary.